common/component/kafka/kafka.go

```diff
@@ -143,6 +143,9 @@ func (k *Kafka) Init(ctx context.Context, metadata map[string]string) error {
 	config.Consumer.Fetch.Default = meta.consumerFetchDefault
 	config.ChannelBufferSize = meta.channelBufferSize
 
+	config.Net.KeepAlive = meta.clientConnectionKeepAliveInterval
+	config.Metadata.RefreshFrequency = meta.clientConnectionRefreshInterval
```

Comment:
the config `config.Metadata.RefreshFrequency` has nothing to do with the tcp connections to kafka, it is a sarama client config to tweak the interval at which the topics metadata is reloaded from the kafka brokers